base/docs/helpdb/Base.jl

```diff
@@ -1927,10 +1927,13 @@ Kronecker tensor product of two vectors or two matrices.
 kron
 
 """
-    randn([rng], [dims...])
+    randn([rng,] [T::Complex128,] [dims...])
```

Comment:
you'd need to update the signatures in the rst docs then run `make docs` to get the docstring content to propagate